### DISCUSSION TITLE
Actual value of requests_max

### DISCUSSION THREAD

**Discussion by pdiblasi** [minio/minio#14337] [Q&A]
Hi all,

when I start minio without setting up `requests_max` parameter, the deafult value is 0, and, according to documentation, when _requests_max_ parameter is set to 0, Minio will automatically calculate it depending on available memory.

_NOTE: A zero value of requests_max means MinIO will automatically calculate requests based on available RAM size and that is the default behavior._
https://github.com/minio/minio/blob/master/docs/throttle/README.md

So, in the startup log on standard output, I see the `requests_max` parameter calculated based on available RAM size

`Automatically configured API requests per node based on available memory on the system: 20`

Now my question is:
Is there a `mc` command, or any other way, to find the actual value of `requests_max`?

Thank you,
Paolo

**Comment by justinmarygopal** [comment]
I am also requiring the same information, do we have any update?
